pkg/kubemark/hollow_proxy.go

```diff
@@ -51,49 +69,56 @@ func NewHollowProxyOrDie(
 	execer utilexec.Interface,
 	broadcaster record.EventBroadcaster,
 	recorder record.EventRecorder,
+	useRealProxier bool,
 ) (*HollowProxy, error) {
-	// Create a proxier with fake iptables underneath it.
-	proxier, err := iptables.NewProxier(
-		iptInterface,
-		sysctl,
-		execer,
-		30*time.Second,
-		5*time.Second,
-		false,
-		0,
-		"10.0.0.0/8",
-		nodeName,
-		getNodeIP(client, nodeName),
-		recorder,
-		nil,
-	)
-	if err != nil {
-		return nil, fmt.Errorf("unable to create proxier: %v", err)
+	// Create a proxier instance.
+	var proxier interface{}
```

Comment:
Done. Took me some time to figure that even pointers can implement interfaces and the interface{} type includes all types (pointers too). Thanks for the pointer ;)